_includes/footer.html

```diff
@@ -1,4 +1,4 @@
-<footer id="footer" class="footer">
+<footer id="footer" class="footer" role="navigation">
```

Comment:
[By default](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/footer), a `<footer>` has  the role [`contentinfo`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Contentinfo_role):

> The contentinfo landmark role is used to identify information repeated at the end of every page of a website, including copyright information, **navigation links**, and privacy statements. This section is commonly called a footer.

(Emphasis mine.)

Thus, I don't think we need this role here:

```suggestion
<footer id="footer" class="footer">
```